wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,73 @@
+# WNT API Examples
+
+This folder contains the Wirepas Network Tool API Examples. These examples show how to interact with
+the server using authentication -, metadata - and real time situation connections.
+
+Please not that these examples should not be taken as production quality, but more
+as the examples about the message flow and content.
+
+## Requirements
+
+Before running the examples please ensure that the 
+[wirepas_messaging](https://github.com/wirepas/backend-apis/tree/master/wrappers/python)
+package is installed.
+
+## WNT backend settings
+
+For running the examples, WNT backend connection parameters needs be given by creating settings.yml
```

Comment:
Change needs to must be given. 

Please provide a clear example on how to run the examples and where the settings.yml should be located. I would also like to see an example of the settings.yml -  What to put in it?

Reply:
Changed "needs" to "must"

Reply:
Example is later in the document